vault/data_source_aws_access_credentials.go

```diff
@@ -155,9 +155,17 @@ func awsAccessCredentialsDataSourceRead(d *schema.ResourceData, meta interface{}
 	d.Set("lease_start_time", time.Now().Format(time.RFC3339))
 	d.Set("lease_renewable", secret.Renewable)
 
+	rootPath := backend + "/config/root"
+	regionData, err := client.Logical().ReadWithData(rootPath, data)
```

Comment:
I may be misunderstanding, but it doesn't look like `data` is actually used here. If that's the case we can just do `client.Logical().Read(rootPath)`.